### ISSUE TITLE
0.9.4: tests fail ("Error: seed index for testdata/blob1 doesn't match its data")

### ISSUE THREAD

**Issue by dvzrv** [folbricht/desync#240]
Hi! I package this project for Arch Linux. When trying to update to 0.9.4 and running tests, I ran into the following issue which prevents me from updating the package:

```
go: downloading github.com/stretchr/testify v1.8.2
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/davecgh/go-spew v1.1.1
Error: chunk id 0589328ff916d08f5fe59a9aa0731571448e91341f37ca5484a85b9f0af14de3 does not match its hash 0000000000000000000000000000000000000000000000000000000000000000
--- FAIL: TestUntarCommandRepair (0.00s)
    untar_test.go:65:
                Error Trace:    github.com/folbricht/desync/cmd/desync/untar_test.go:65
                Error:          Error message not equal:
                                expected: "chunk id 0589328ff916d08f5fe59a9aa0731571448e91341f37ca5484a85b9f0af14de3 does not match its hash c672b8d1ef56ed28ab87c3622c5114069bdd3ad7b8f9737498d0c01ecef0967a"
                                actual  : "chunk id 0589328ff916d08f5fe59a9aa0731571448e91341f37ca5484a85b9f0af14de3 does not match its hash 0000000000000000000000000000000000000000000000000000000000000000"
                Test:           TestUntarCommandRepair
Error: seed index for testdata/blob1 doesn't match its data
FAIL
FAIL    github.com/folbricht/desync/cmd/desync  7.774s
FAIL
```

You can find the build script for 0.9.3 [here](https://gitlab.archlinux.org/archlinux/packaging/packages/desync/-/blob/main/PKGBUILD).

**Comment by folbricht** [comment]
Thanks for reporting this. I fixed the test and made v0.9.5 with the fix. Hope that works

**Comment by dvzrv** [comment]
> Thanks for reporting this. I fixed the test and made v0.9.5 with the fix. Hope that works

Thanks! Yes, that fixed it! :tada:
